skimage/filters/tests/test_thresholding.py

```diff
@@ -570,6 +570,8 @@ def test_mean():
 
 
 def test_triangle_uint_images():
+    assert(threshold_triangle(np.zeros((10, 10), dtype='uint8')) == 0)
+    assert(threshold_triangle(np.zeros((10, 10))) == 0.0)
```

Comment:
Might be a good idea to move this into it's own test with a parametrization for different types of uniform arrays and data types...